.surge.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -x
+
+if [ "$TRAVIS_EVENT_TYPE" = "push" -a "$TRAVIS_BRANCH" = "master" ]; then
+  docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" -e SURGE_LOGIN="$SURGE_LOGIN" -e SURGE_TOKEN="$SURGE_TOKEN" yast-ycp-ui-bindings-image /bin/bash -c "zypper  in -y npm doxygen; npm install --global surge; rake doc; surge --project ./autodoc/html --domain yast-ui-bindings.surge.sh"
```

Comment:
NP: Also think about forks, if you enable Travis for your fork it won't work.

- By default `$SURGE_LOGIN` `$SURGE_TOKEN` will be undefined, resulting in invalid credentials
- Even if you provide your credentials it won't work as the `yast-ui-bindings.surge.sh` domain is blocked for the original account.

You should probably enable the deployment only in the original `yast/yast-ycp-ui-bindings` GitHub repo. (See [how to check the Github repo](https://github.com/yast/yast.github.io/blob/master/.travis_deploy.rb#L37).)